vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -24,6 +24,12 @@ do
     fi
 done
 
+# generate Windows image version during offical build which will be shared between PIR and SIG
+if [[ ${OS_NAME} == "Windows" ]]; then
```

Comment:
We'll benefit from this when publishing the image the different locations, in our case, PIR and SIG. so that folks don't have to search and manually specify the version for images